networking/src/ApiClient.ts

```diff
@@ -885,10 +882,11 @@ class QuestionTemplateParser {
             version,
             choices,
             instructions,
-            domain,
-            cluster,
-            grade,
-            standard,
+            answerSettings,
+            domain: domain ?? null,
+            cluster: cluster ?? null,
+            grade: grade ?? null,
+            standard: standard ?? null,
```

Comment:
Is this for accounting for `undefined`?

Reply:
`null` and `undefined` have been removed from the type per previous comments. Therefore `?? null` has been removed